### ISSUE TITLE
max depth still 8000 with -d set up

### ISSUE THREAD

**Issue by trptyrphe** [genome/bam-readcount#19]
I installed by git clone and it runs well. I found that max count per position is still 8000 even I set up -d equals to 1000,000. A bug or wrong command line? Thanks.

**Comment by ernfrid** [comment]
How are you trying to run the software? I did glance at the code and it looks like there is likely a bug with that parameter for the case where no region or region list is specified.

**Comment by trptyrphe** [comment]
Yes I run without providing a region. And -d would work if I add the region.

**Comment by chadisaad** [comment]
Still have the bug

**Comment by jakubdolata86** [comment]
Hello, I'm using 0.8.0 version and I have the same problem as above... maximum depth 8000. How to fix it?

Jakub
